rules/sinks/third_parties/sdk/openai/python.yaml

```diff
@@ -11,4 +11,6 @@ sinks:
     patterns:
       - "(?i)(openai|chronological).*"
       - "(?i)langchain[.]llms.*(openai|azureopenai|openllm).*"
+      - "(?i).*(openai.ChatCompletion).*"
+      - "(?i)(langchain.chat_models).*(ChatOpenAI).*"
```

Comment:
Both the new regex are already covered by `openai.*` pattern we have above

Reply:
No, In new pattern we are matching methodFullName which may not start with  `openai` but includes `openai.chatcomletion`